examples/00_getting_started.py

```diff
@@ -0,0 +1,231 @@
+"""
+Getting Started
+===============
+
+The purpose of this guide is to provide an introduction to the functionalities of
+``skrub``, an open-source package that aims at bridging the gap between tabular
+data sources and machine-learning models. Please refer to our `installation guidelines
+<https://skrub-data.org/stable/install.html>`_ for installing ``skrub``.
+
+Much of ``skrub`` revolves around vectorizing, assembling, and encoding tabular data,
+to prepare data in a format that machine-learning models understand.
+"""
+
+# %%
+# Downloading example datasets
+# ----------------------------
+#
+# The :obj:`~skrub.datasets` module allows us to download tabular datasets and
+# demonstrate ``skrub``'s features.
+
+# %%
+from skrub.datasets import fetch_employee_salaries
+
+dataset = fetch_employee_salaries()
+employees_df, salaries = dataset.X, dataset.y
+
+# %%
+# Explore all the available `datasets
+# <https://skrub-data.org/stable/reference/downloading_a_dataset>`_.
+
+
+# %%
+# Generating an interactive report for a dataframe
+# -------------------------------------------------
+#
+# To quickly get an overview of a dataframe's contents, use the
+# :class:`~skrub.TableReport`.
+
+# %%
+from skrub import TableReport
+
+TableReport(employees_df)
+
+# %%
+# You can use the interactive display above to explore the dataset visually.
+
+# %%
+# Easily building a strong baseline for tabular machine learning
+# --------------------------------------------------------------
+#
+# The goal of ``skrub`` is to ease tabular data preparation for machine learning.
+# The :func:`~skrub.tabular_learner` function provides an easy way to build a simple
+# but reliable machine-learning model, working well on most tabular data.
+
+
+# %%
+from sklearn.model_selection import cross_validate
+
+from skrub import tabular_learner
+
+model = tabular_learner("regressor")
+results = cross_validate(model, employees_df, salaries)
+results["test_score"]
+
+# %%
+# To handle rich tabular data and feed it to a machine-learning model, the
+# pipeline returned by :func:`~skrub.tabular_learner` preprocesses and encodes
+# strings, categories and dates using the :class:`~skrub.TableVectorizer`.
+# See its documentation or :ref:`sphx_glr_auto_examples_01_encodings.py` for
+# more details. An overview of the chosen defaults is available in
+# `end-to-end pipeline <https://skrub-data.org/stable/end_to_end_pipeline>`_.
```

Comment:
this link is also broken, and if possible make it a relative link (use a sphinx :ref: directive rather than a hyperlink)